src/lib/utils/component.ts

```diff
@@ -179,6 +179,10 @@ export abstract class AbstractComponent<O extends IComponentHost> extends EventD
         return this._componentName;
     }
 
+    set componentName(name: string) {
+        this._componentName = name;
+    }
```

Comment:
Why not just turn this into a property instead of a getter and setter that does the same thing?